operator/v1/types_network.go

```diff
@@ -209,3 +283,29 @@ const (
 	// for sophisticated network isolation and segmenting. This is the default.
 	SDNModeNetworkPolicy SDNMode = "NetworkPolicy"
 )
+
+// MacvlanMode is the Mode of macvlan. The value are lowercase to match the CNI plugin
+// config values. See "man ip-link" for its detail.
+type MacvlanMode string
+
+const (
+	// MacvlanModeBridge is the macvlan with thin bridge function.
+	MacvlanModeBridge MacvlanMode = "bridge"
+	// MacvlanModePrivate
+	MacvlanModePrivate MacvlanMode = "private"
+	// MacvlanModeVEPA is used with Virtual Ethernet Port Aggregator
+	// (802.1qbg) swtich
+	MacvlanModeVEPA MacvlanMode = "vepa"
+	// MacvlanModePassthru
+	MacvlanModePassthru MacvlanMode = "passthru"
+)
+
+// IPAMType describes the IP address management type to configure
+type IPAMType string
+
+const (
+	// IPAMTypeDHCP uses DHCP for IP management
+	IPAMTypeDHCP IPAMType = "dhcp"
```

Comment:
Likewise, this should be "DHCP" and "Static"

Reply:
changed.